src/components/HeaderWithLanguage/HeaderStatusInformation.jsx

```diff
@@ -118,6 +121,7 @@ const HeaderStatusInformation = ({
         </StyledStatus>
         {published && (taxonomyPaths?.length > 0 ? publishedIconLink : publishedIcon)}
         {multipleTaxonomyIcon}
+        {learningpathConnections}
         {helperIcon}
```

Comment:
Egentlig kunne vi fjerna dette ikonet fordi når du trykker på det får du kun en dialog uten innhold som gir meining.

Reply:
`helperIcon`?

Reply:
Kunne endra dette til å sjå feks slik ut
![image](https://user-images.githubusercontent.com/8956884/112156712-ccd23800-8be6-11eb-9ebb-6c47c90e1c56.png)

Reply:
Ja. Fjerne helperIcon og sette dei andre ikona på andre sida av status. Men kun en splitter for ellers blir det litt rart for nye artikler som ikkje har statusikon.

Reply:
Ok så treng ikkje læringssti-ikonet endre farge. Kan være primary heile tida slik at den står i stil med dei andre ikona som ikkje endrer farge on focus.

Reply:
Dette gjelder alle ikonene i den boksen Gunnar screenshotter: jeg klarer ikke tabbe meg til dem. Ganske sikker på at det er et UU-krav? Klarer du å fikse det eller bør vi opprette en ny issue på det? 👼🏼

Reply:
Prøvde nå, og som du sier tabber den bare til det som er definert som knapper eller linker. Så feks når en artikkel har flere oppføringer i taksonomi, kan man tabbe til check-ikonet, for da er den wrappa med en <safelink>. Så la bare til en knapp rundt læringssti-ikonet sånn at den også blir tilgjengelig!

Men når ikonene ikke har en link blir de altså ikke tilgjengelig fra tab, og jeg prøvde også å kjøre med VoiceOver på siden, og de blir ikke tilgjengelig fra den heller. Så det er jo litt dumt. Men det er jo dumt å gjøre dem til en knapp, så kanskje det må gjøres noe for at ikoner med Tooltip hvertfall er tilgjengelig med sånne Voiceover-verktøy?